Source/Csla/ApplicationContext.cs

```diff
@@ -26,12 +27,17 @@ public class ApplicationContext
     /// Creates a new instance of the type
     /// </summary>
     /// <param name="applicationContextAccessor"></param>
-    public ApplicationContext(ApplicationContextAccessor applicationContextAccessor)
+    /// <param name="options"></param>
+    public ApplicationContext(ApplicationContextAccessor applicationContextAccessor, CslaOptions options)
```

Comment:
As already mentioned in discord.
This is technically a breaking change. We should implement it this way in v10. But here we must use `ApplicationContextAccessor.ServiceProvider.GetRequiredService<CslaOptions>` to avoid the breaking change.